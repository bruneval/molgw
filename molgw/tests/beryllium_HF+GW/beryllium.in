&molgw
 scf='hf'
 basis='cc-pVDZ'
 basispath='../../basis'
 postscf='gw'
 mixing_scheme='simple'
 natom=1
/
Be 0. 0. 0.
