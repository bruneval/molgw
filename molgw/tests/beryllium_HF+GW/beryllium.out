 ======================================================================


             Welcome to the fascinating world of MOLGW


 ======================================================================

 Today is 02/27/2015
 It is now 12:16
 Go and get some good food
 Compilation options

 WARNING: LIBXC version  2. 2

 WARNING: calculation name: HF+GW

 Summary of the input parameters 

       Calculation type:   HF+GW                                                                                               
               SCF type:   HF                                                                                                  
          Post SCF type:   GW                                                                                                  
                  Natom:   1
              Electrons:   4.0000
                 Charge:   0.0000
          Magnetization:   0.0000
              Basis set:   cc-pVDZ                                                                                             
    Auxiliary basis set:                                                                                                       
          Gaussian type:   PURE
      Spin polarization:   1
              SCF steps:  30
                 Mixing:   0.7000
           Grid quality:   HIGH                                                                                                
       Integral quality:   HIGH                                                                                                

        IO options:
        - matrices details:     F
        - basis set details:    F
        - ERI file:             F
        - ignore big RESTART:   F
        - plot some wfns:       F
        - dump spectral functs  F

 ================================
       atom list
                        bohr                                        angstrom
  Be     0.000000     0.000000     0.000000           0.000000     0.000000     0.000000
 ================================
 Number of bonds     0
 Molecule has inversion symmetry
 Molecule is linear


                  Total number of basis functions:      14
                  Total number of cart. functions:      15
                Total number of diffuse functions:       0
                                 Number of shells:       6
        Maximum angular momentum in the basis set:       2
                                                         d
 Basis set is ready and fit

 Setting up the cartesian to pure transforms
 Transformations set up completed

 Setup overlap matrix S

 Tolerance on integrals set to  1.00E-10

 Cauchy-Schwartz screening of the 4-center integrals
 Libint library initialized
 Max angular momentum handled by your Libint compilation:     7


 Non negligible shellpairs to be computed      21  over a total of      21

 Calculate and store all the Electron Repulsion Integrals (ERI)
 Libint library initialized
 Max angular momentum handled by your Libint compilation:     7


 Number of integrals to be stored:        5565
 Max index size  2147483647
 Allocate 4-center integrals
 Memory [Mb]:     0.042
 success
 All ERI have been calculated


 === Occupations ===
   1      2.00000
   2      2.00000
   3      0.00000
   4      0.00000
   5      0.00000
   6      0.00000
   7      0.00000
   8      0.00000
   9      0.00000
  10      0.00000
  11      0.00000
  12      0.00000
  13      0.00000
  14      0.00000


 WARNING: Restart from a big RESTART file obtained within HF

 Try to read spectral function from file spectral_file
 File does not exist


 Calculating the polarizability
 Singlet state

 Build the electron-hole hamiltonian
 Build Common part: Energies + Hartree + possibly Exchange
 Content of Exchange:    0.000

 Performing the block diago with square root of matrices
 Diago to get (A - B)^{1/2}                         24 x       24
 Diago (A - B)^{1/2} * (A + B) * (A - B)^{1/2}      24 x       24

 Calculate the RPA energy using the Tamm-Dancoff decomposition
 Eq. (9) from J. Chem. Phys. 132, 234114 (2010)

 RPA energy [Ha]:    -0.04136298

 Lowest neutral excitation energy [eV]   12.23552412

 Build W = v * chi * v

 Spectral function initialized with npoles                 :       48
 Spectral function initialized with Coulomb basis functions:      105
 Memory [Mb]:     0.077


 RPA Total energy [Ha]:    -0.0413629803

 WARNING: small complex number is  1.00E-03

 perform a one-shot G0W0 calculation

 G0W0 Eigenvalues [eV]
   #          E0        Sigx-Vxc      Sigc          Z         G0W0
   1   -128.773149     0.000000     2.573291     0.888132  -126.487726
   2     -8.409367     0.000000    -0.610833     0.949590    -8.989408
   3      1.585302     0.000000    -0.384931     0.972418     1.210988
   4      1.585302     0.000000    -0.384931     0.972418     1.210988
   5      1.585302     0.000000    -0.384931     0.972418     1.210988
   6      8.131379     0.000000    -0.784878     0.940536     7.393173
   7      9.528879     0.000000    -1.097483     0.929857     8.508377
   8      9.528879     0.000000    -1.097483     0.929857     8.508377
   9      9.528879     0.000000    -1.097483     0.929857     8.508377
  10     17.708444     0.000000    -0.679972     0.697947    17.233859
  11     17.708444     0.000000    -0.679972     0.697947    17.233859
  12     17.708444     0.000000    -0.679972     0.697947    17.233859
  13     17.708444     0.000000    -0.679972     0.697947    17.233859
  14     17.708444     0.000000    -0.679972     0.697947    17.233859

 Writing energy_qp file

 Spectral function destroyed

                 --- Timings in (s) and # of calls ---

                    Total time              0.17

                 ----------------------

                 Total pre SCF              0.16
                     Total SCF              0.00
                Total post SCF              0.00

                 ----------------------

            4-center integrals              0.14         1
           Kinetic Hamiltonian              0.00         0
   Electron-nuclei Hamiltonian              0.00         0

                       Hartree              0.00         0
                      Exchange              0.00         0
                        DFT xc              0.00         0
             Hamiltonian diago              0.00         0
          RESTART file writing              0.00         0

                 Single Excit.              0.00         0

        Total chi polarization              0.00         1
             ERI basis transform            0.00         6
              Build 2 particle H            0.00         1
                       Common part          0.00         1
                        TDDFT part          0.00         0
                          BSE part          0.00         0
              Diago 2 particle H            0.00         1
             Invert 2 particle S            0.00         0
                         Build W            0.00         1
                Optical spectrum            0.00         0

                GW self-energy              0.00         1

                    MP2 energy              0.00         0
               MP2 self-energy              0.00         0
                 ----------------------

 SUMMARY of all the WARNINGS issued during the run:

 1.     LIBXC version  2. 2                                                                                 
 2.     calculation name: HF+GW                                                                             
 3.     Restart from a big RESTART file obtained within HF                                                  
 4.     small complex number is  1.00E-03                                                                   



 This is the end

